src/main/windows/main.ts

```diff
@@ -51,37 +51,31 @@ export function init(options: { hidden: boolean }) {
 
   const isMac = platform() === 'darwin'
 
-  const main_window = (window = <ExtendedBrowserWindow>(
-    new electron.BrowserWindow({
-      backgroundColor: '#282828',
-      // backgroundThrottling: false, // do not throttle animations/timers when page is background
-      darkTheme: true, // Forces dark theme (GTK+3)
-      icon: appIcon(),
-      show: false,
-      title: appWindowTitle,
-      height: initialBounds.height,
-      width: initialBounds.width,
-      x: initialBounds.x,
-      y: initialBounds.y,
-      webPreferences: {
-        nodeIntegration: false,
-        preload: defaults.preload,
-        spellcheck: false, // until we can load a local dictionary, see https://github.com/electron/electron/issues/22995
-        webSecurity: true,
-        allowRunningInsecureContent: false,
-        contextIsolation: false,
-      },
-      titleBarStyle: isMac ? 'hidden' : 'default',
-      titleBarOverlay: true,
-    })
-  ))
-  main_window.filePathWhiteList = []
-
-  initMinWinDimensionHandling(
-    main_window,
-    defaults.minWidth,
-    defaults.minHeight
-  )
+  const _window = (window = <ExtendedBrowserWindow>new electron.BrowserWindow({
```

Comment:
why not rename it to `mainWindow`? `_` makes me wonder wether it's unused or private?